build/Dockerfrag.kics

```diff
@@ -5,6 +5,10 @@ ENV KICS_INCLUDE_QUERIES_PATH="/home/easy_infra/.kics/assets/queries"
 ENV KICS_LIBRARY_PATH="/home/easy_infra/.kics/assets/libraries"
 ENV KICS_JSON_REPORT_PATH="/tmp/reports/kics"
 
-COPY --from=kics --chown=easy_infra:easy_infra /usr/local/bin/kics /usr/local/bin/kics
-COPY --from=kics --chown=easy_infra:easy_infra /home/easy_infra/.kics /home/easy_infra/.kics
+COPY --from=kics --chown=easy_infra:easy_infra /app/bin/kics /usr/local/bin/kics
+COPY --from=kics --chown=easy_infra:easy_infra /root/.kics /home/easy_infra/.kics
 COPY --from=kics --chown=easy_infra:easy_infra /tmp/reports/kics /tmp/reports/kics
+
+# Intentionally left out because KICS is not currently used in the Terraform image
```

Comment:
When KICS went from 1.5.1 to 1.5.2+ and removed release binaries they said it was due to local environmental requirements per https://github.com/Checkmarx/kics/discussions/4744.  After reviewing their dockerfile, it seems those local environmental requirements only have to do with terraform scans, which we don't use, but I did want to document it for posterity.